### PR TITLE
MAT-1135: adding checks for secret files

### PR DESCRIPTION
**Pull request by tokamak-git**
adding check for secret files. it shall log an error if permissions are not set as expected.
added a file permission checker func. it is called in the bridge start.

### CODE REVIEW

**Comment by tokamak-git** [comment]
addressed

File: file/permCheck.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package file
```
Comment (jdkanani) [inline file/permCheck.go:1]: On file names, shouldn't we use `_` or `-` separation for file names?

Comment (tokamak-git) [reply file/permCheck.go]: using snake case for file names

File: types/error/errInvalidPermissions.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package types
```
Comment (jdkanani) [inline types/error/errInvalidPermissions.go:1]: Please rename filename to `invalid_permissions.go`, just to follow filename conventions.

**Comment by tokamak-git** [comment]
@mankenavenkatesh hey please let me know if I can go ahead with merging this. have implemented the changes you requested.

**Pull request merged**